README.md

```diff
@@ -1,3 +1,106 @@
 # not911.me
 
-Alternatives to 911 in the Seattle area.
+Alternatives to dialing 911.
+
+## To submit a correction or add a new resource
+
+Either open a pull request with the correction/new resource or [open an issue](https://github.com/Automattic/wp-calypso/issues/new/choose).
+
+If neither of those options work for you, you can also securely email us at <techblocsea@protonmail.com>.
+
+## Development
+
+To run the site locally:
+
+1. `gem install bundler`
+1. `bundle install`
+1. `bundle exec jekyll serve` -> This will run the site at `localhost:4000`
+1. Make changes and contribute 🙌
+
+## Deployment
+
+The site is currently deployed through Netlify. This gives us a lot of good stuff like free deployment previews on pull requests as well as the contact form.
+
+To deploy the site, simply merge a PR and Netlify will take care of the rest.
+
+## How this site is structured
+
+### Cities
+
+This site is organized into several layers, the topmost being the "city" layer. A city is described by a simple markdown file with YAML frontmatter. For example, here's a minimal version of the Seattle, WA `city` markdown file:
+
+```markdown
+---
+title: Seattle, WA
+slug: seattle
+nonemergency_phone:
+  link: 2066255011
+  display: (206) 625-5011
+---
+```
+
+All these fields are required for a city.
+
+You may also optionally add a `prior_art` property which can be any valid HTML. This will be rendered in the About section for the city. [See the actual Seattle city file for an example of this](https://raw.githubusercontent.com/SeattleDSA/not911.me/HEAD/_cities/seattle.md)
+
+Note that the content of a city markdown file is completely ignored, so don't bother putting anything there.
+
+### Resource Types
+
+The next layer is the "resource type" layer. These represent the various categories of resources that are displayed on each city's page. They are yet again simply a markdown file with YAML frontmatter. This is a minimal example:
+
+```markdown
+---
+title: Housing
+slug: housing
+order: 0
+---
+```
+
+The resource type buttons and the resources themselves are ordered on the city page based on the `order` attribute of the resource type.
```

Comment:
Just curious, do we know what happens when two resource types are given the same order?

Reply:
I think they'll be ordered by the way the file system orders them, at least that's the default ordering. Hopefully we won't have that though.

I actually considered adding a 0 to all the existing orderings to make it easier to add categores inbetween the existing ones. What do you think?

Reply:
If all are 0 though doesn't that mean we won't be able to chose the ordering?

Reply:
Oh no, _adding_ a 0, like `10` instead of `1`, `20` instead of `2`. That makes it possible to add like, `11` or `14` to add something between `10` and `20`. I guess we can use decimals for that now though 🤷

Reply:
Oh!! Got it, my bad. Yea both sound fine!